services/server/src/server/services/workers/workerTypes.ts

```diff
@@ -36,6 +37,14 @@ export interface VerifyFromEtherscanInput extends VerificationWorkerInput {
   etherscanResult: EtherscanResult;
 }
 
+export interface VerifySimilarityInput extends VerificationWorkerInput {
+  chainId: string;
+  address: string;
+  runtimeBytecode: string;
+  creatorTxHash?: string;
```

Comment:
```suggestion
  creationTransactionHash?: string;
```

For consistency, the other interfaces of the worker use this naming